### PR TITLE
0.1.1 24 local_data_platform.target.local.iceberg.put(dataframe)

### PR DESCRIPTION
**Pull request by tusharchou**
Given a parquet of taxi rides for NYC yellow taxi datasets read the parquet in a class called SourceParquetTable.read()

### CODE REVIEW

**Comment by tusharchou** [comment]
@mrutunjay-kinagi coming up iceberg table class

File: local-data-platform/nyc_yellow_taxi.py [hunk 1]
```diff
@@ -23,3 +24,23 @@
 
 # Verify if the catalog is set up correctly
 print("Catalog set up successfully:", catalog)
+
+'''
+CLI COMMAND to grab data -
```
Comment (mrutunjay-kinagi) [inline local-data-platform/nyc_yellow_taxi.py:29]: Is this for local testing ?

**Review by mrutunjay-kinagi — DISMISSED** [review]
Looks good to me !

File: local-data-platform/nyc_yellow_taxi.py (hunk 1 above)
Comment (tusharchou) [reply local-data-platform/nyc_yellow_taxi.py:29]: @mrutunjay-kinagi yes. I ll added it to the modules read me like iceberg.

**Review by mrutunjay-kinagi — DISMISSED** [review]

**Comment by tusharchou** [comment]
@mrutunjay-kinagi need a fresh review again

File: local-data-platform/nyc_yellow_taxi.py [hunk 2]
```diff
@@ -1,25 +1,67 @@
...
+from local_data_platform.target.iceberg.table import IcebergTable
+from local_data_platform.source.parquet.pyarrow_table import PyarrowTable
+from local_data_platform import BaseConfig
+
 
 config = {
-    "name": "NYC Yellow Taxi",
-    "path": "/Users/tushar/Documents/GitHub/local-data-platform/sample_data/parquet/nyc_yellow_taxi/2023-01/yellow_tripdata_2023-01.parquet"
+    "name": "nyc_yellow_taxi_rides",
+    "path": "/Users/tushar/Documents/GitHub/local-data-platform/local-data-platform/yellow_tripdata_2023-01.parquet",
```
Comment (mrutunjay-kinagi) [inline local-data-platform/nyc_yellow_taxi.py:11]: Use .env or add it to the codebase.

File: local-data-platform/local_data_platform/__init__.py
Comment (mrutunjay-kinagi) [inline local-data-platform/local_data_platform/__init__.py:1]: This looks interesting. What design approach you are implementing here ?

**Review by mrutunjay-kinagi — APPROVED** [review]

**Review by redpheonixx — COMMENTED** [review]
are these only skeleton changes? can we have KT to understand the structure and how to deploy and work on local system.

**Review by mrutunjay-kinagi — APPROVED** [review]
Approved

**Pull request merged**